iOS/Flows/Conversation/Cells/TimeMachineCollectionViewLayout.swift

```diff
@@ -53,21 +53,14 @@ class TimeMachineCollectionViewLayout: UICollectionViewLayout {
     // MARK: - Layout Configuration
 
     /// The height of the cells.
-    var itemHeight: CGFloat
-    = MessageContentView.bubbleHeight + MessageDetailView.height + Theme.ContentOffset.short.value {
-        didSet { self.invalidateLayout() }
-    }
+    var itemHeight: CGFloat = 100
     /// Keypoints used to gradually shrink down items as they move away.
     var scalingKeyPoints: [CGFloat] = [1, 0.84, 0.65, 0.4]
     /// The amount of vertical space between the tops of adjacent items.
-    var spacingKeyPoints: [CGFloat] {
-        switch self.uiState {
-        case .read:
-            return [0, 18, 22, 26]
-        case .write:
-            return [0, 8, 16, 20]
-        }
-    }
+    var spacingKeyPoints: [CGFloat] = [0, 8, 16, 20]
+    var firstSectionTopY: CGFloat = 0
+    var secondSectionBottomY: CGFloat = 300
```

Comment:
This lets you more precisely position where the message sections appear on screen.